app/helpers/planning_data_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module PlanningDataHelper
+  PLANNING_DATA_BASE_URL = "https://www.planning.data.gov.uk/"
+
+  def planning_data_entity_url(entity_id)
+    "#{PLANNING_DATA_BASE_URL}entity/#{entity_id}"
+  end
+
+  def planning_data_map_url(datasets, planning_application)
+    base_url = "#{PLANNING_DATA_BASE_URL}map/"
+    return base_url unless datasets.any?
+
+    uri = URI.join(PLANNING_DATA_BASE_URL, "map/")
+    uri.query = URI.encode_www_form({dataset: datasets.flatten})
+    uri.fragment = lat_lon_zoom(planning_application)
```

Comment:
Would we maybe want to include the lat/long even if there are no datasets?

```suggestion

    uri = URI.join(PLANNING_DATA_BASE_URL, "map/")
    uri.query = URI.encode_www_form({dataset: datasets.flatten}) if datasets.any?
    uri.fragment = lat_lon_zoom(planning_application)
```

Reply:
Yeah good call